
{'date': 'Tue Oct 31 2023 15:13:14.308 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    Pokebot()
  module <module> line 20
    ppp = window.puppeteer
AttributeError: no attribute puppeteer for [object Window]
'''},
{'date': 'Tue Oct 31 2023 15:46:39.163 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 32
  with urllib.request.urlopen(url) as _fp
                                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Oct 31 2023 15:47:23.473 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 32
  with urllib.request.urlopen(url) as _fp
                                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Tue Oct 31 2023 15:47:44.3 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 33
  _data = _fp.read()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Tue Oct 31 2023 15:48:02.853 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <NetworkError: A network error occurred.>
'''},
{'date': 'Tue Oct 31 2023 15:48:51.98 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <NetworkError: A network error occurred.>
'''},
{'date': 'Tue Oct 31 2023 15:52:36.341 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 42
    data = parser.url_get("https://pokemon.fandom.com/pt-br/wiki/Pok%C3%A9dex_Nacional")
  module <module> line 32
    _fp = urllib.request.urlopen(src)
NameError: name 'src' is not defined
'''},
{'date': 'Tue Oct 31 2023 15:52:48.462 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <NetworkError: A network error occurred.>
'''},
{'date': 'Tue Oct 31 2023 16:37:42.569 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <NetworkError: A network error occurred.>
'''},
{'date': 'Tue Oct 31 2023 17:07:49.29 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 47
    parser = Pokebot()
TypeError: __init__() missing 1 positional argument: url
'''},
{'date': 'Tue Oct 31 2023 17:08:09.569 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 50
    parser.feed('''<html><head><title>Test</title></head>
  module html.parser line 144
    self. rawdata=self. rawdata+data
AttributeError: 'Pokebot' object has no attribute 'rawdata'
'''},